swim/handlers.go

```diff
@@ -121,9 +121,22 @@ func (n *Node) tickHandler(ctx json.Context, req *emptyArg) (*ping, error) {
 	return &ping{Checksum: n.memberlist.Checksum()}, nil
 }
 
-func (n *Node) adminJoinHandler(ctx json.Context, req *emptyArg) (*Status, error) {
-	n.memberlist.MakeAlive(n.address, util.TimeNowMS())
-	return &Status{Status: "rejoined"}, nil
+type Clock interface {
```

Comment:
I feel uneasy at adding a Clock interface in the middle of `swim/handlers.go` -- my feeling is that if we add a mechanism for mock/virtual clocks, that it should come from the top -- Ringpop->Node->handlers.

I recognize that it's a localized problem atm... but don't like the queasy feeling.

@jwolski @CorgiMan whatcha think? too much?